README.rst

```diff
@@ -143,7 +143,7 @@ Configuration
    ::
 
        $ cp node-agent/etc/tendrl/node-agent/logging.yaml.timedrotation.sample \
-         /etc/tendrl/node-agent_logging.yaml
```

Comment:
- Delete the file `logging.yaml.timedrotation.sample`

- Use `logging.yaml.syslog.sample`